invenio_opendata/base/recordext/fields/opendata.cfg

```diff
@@ -132,15 +132,15 @@ formatted_contents_note:
     producer:
         json_for_marc(), {"505__g": "miscellaneous", "516__t": "title"}
 
-# 516__a 516__w
+# 516__a 516__u 516__w 516__y
 
 computer_file_type:
     schema:
         {'computer_file_type': {'type': 'list', 'force': True}}
     creator:
-        marc, "516__", {'note': value['a'], 'recid': value['w']}
+        marc, "516__", {'note': value['a'], 'url': value['u'], 'recid': value['w'], 'description': value['y'}
     producer:
-        json_for_marc(), {"516__a": "note", "516__w": "recid"}
+        json_for_marc(), {"516__a": "note", "516__u": "url", "516__w": "recid" "516__y": "description"}
```

Comment:
If you are deleting 516 $w, you have to be fully sure that we don't have any such subfield left... It's always safer to just keep the old definitions (if they are good) and only add new ones subfields that we might be missing.